File: API/ModelsDTO/LoginDTO.cs
using API.Enums;

namespace API.ModelsDTO
{
    public class LoginDTO
    {
        public string Login { get; set; }
        public string Password { get; set; }
        public string AccessToken { get; set; }
        public string RefreshToken { get; set; }
        public GrantType GrantType { get; set; }
    }
}

File: API/ClientApp/src/app/models/LoginDTO.ts
import { GrantType } from "./Enums/GrantType";

export class LoginDTO {

    constructor() {
        this.login = "";
        this.password = "";
        this.grantType = GrantType.Password;
    }

    login: string;
    password: string;
    grantType: GrantType;
    accessToken?: string = "";
    refreshToken?: string = "";
}